dev/io.openliberty.netty.internal.impl/bnd.bnd

```diff
@@ -0,0 +1,64 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Liberty Netty Transport Implementation
+Bundle-SymbolicName: io.openliberty.netty.internal.impl
+Bundle-Description: Liberty Netty Transport Implementation ${bVersion}
+WS-TraceGroup: Netty
+
+Import-Package: \
+  io.openliberty.netty.internal,\
+  *
```

Comment:
Your choice, but might be better as
	${defaultPackageImport}
?

Reply:
If you do go back in, a lot of them have the Export-Package before the imports but again, your choice.